code/datums/jobs.dm

```diff
@@ -1227,11 +1227,13 @@ ABSTRACT_TYPE(/datum/job/civilian)
 /datum/job/special/research_assistant
 	name = "Research Assistant"
 	linkcolor = "#9900FF"
-	limit = 1
+	limit = 2
 	wages = PAY_UNTRAINED
 	low_priority_job = 1
-	slot_jump = list(/obj/item/clothing/under/color/white)
-	slot_foot = list(/obj/item/clothing/shoes/white)
+	slot_jump = list(/obj/item/clothing/under/color/lightpurple)
+	slot_foot = list(/obj/item/clothing/shoes/black)
```

Comment:
I feel like white shoes would fit better as that's what scientists have but also no strong opinions really.